tests/eval_files/trio102.py

```diff
@@ -162,6 +162,55 @@ async def foo3():
             await foo()  # safe in theory, error: 12, Statement("try/finally", lineno-12)
 
 
+async def foo4():
+    try:
+        ...
+    except ValueError:
+        await foo()  # safe
+    except BaseException:
+        await foo()  # error: 8, Statement("BaseException", lineno-1)
+    except:
+        await foo()  # safe, since BaseException will catch Cancelled
```

Comment:
...safe, yes, but also redundant with the `except BaseException:` 🤔

Reply:
I didn't actually know that exceptions *have* to inherit from BaseException, interesting. I opened https://github.com/PyCQA/flake8-bugbear/issues/349 in case anybody cares for it.